deb/env.txt

```diff
@@ -0,0 +1,6 @@
+JJS_SYSROOT=/var/lib/jjs
+JJS_PATH=/usr
```

Comment:
/opt/jjs

Reply:
What's wrong with installing into /usr?

Reply:
Because it is wasting OS with rubbish like `/usr/etc/example-config`